Dockerfile.builder

```diff
@@ -17,43 +14,35 @@ ENV PATH "$GOROOT/bin:$GOPATH/bin:$PATH"
 ENV GO_VERSION 1.14.1
 
 #
-# Build dependencies
-#
-RUN apt-get update \
-    && DEBIAN_FRONTEND=noninteractive apt-get upgrade -y --no-install-recommends \
-	&& DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-		apt-utils \
-		binutils \
-		ca-certificates \
-		clang-7 \
-		coreutils \
-		curl \
-		gcc \
-		git \
-		iproute2 \
-		libc6-dev \
-		libc6-dev-i386 \
-		libelf-dev \
-		llvm-7 \
-		m4 \
-		make \
-		pkg-config \
-		python \
-		rsync \
-		unzip \
-		wget \
-		zip \
-		zlib1g-dev \
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
-	&& update-alternatives --install /usr/bin/clang clang /usr/bin/clang-7 100 \
-	&& update-alternatives --install /usr/bin/llc llc /usr/bin/llc-7 100
+# Basic build dependencies
+#
+RUN \
+apt-get update && \
+apt-get upgrade -y --no-install-recommends && \
+apt-get install -y --no-install-recommends \
+# Base Cilium-build dependencies
+  binutils \
+  coreutils \
+  curl \
+  gcc \
+  git \
+  libc6-dev \
+  libelf-dev \
+  make \
+  rsync \
+  unzip \
+  wget \
+  zip && \
```

Comment:
Are these still needed? Would it make sense to document for each of of the installed packages why they're needed?

Reply:
Not 100% sure whether some of the above are still needed. I guess I'll find out via trial and error for some of them. ;) I'll document the ones needed, like in cilium-runtime, agree.